buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -84,7 +84,7 @@ class PrecommitTasks {
         Task mainForbidden = project.tasks.findByName('forbiddenApisMain')
         if (mainForbidden != null) {
             mainForbidden.configure {
-                signaturesURLs += getClass().getResource('/forbidden/es-core-signatures.txt')
+                signaturesURLs += getClass().getResource('/forbidden/es-server-signatures.txt')
```

Comment:
Is this name change right? We do want these signatures forbidden basically everywhere?

Reply:
The problem at the moment is that there are fobidden things in those files that are dependencies of server but not `elasticsearch-core`. So I felt that naming it `server` right now was more accurate.

Reply:
The problem is that there are forbidden things in that file are dependencies of `server` but not `elasticsearch-core`. So I felt that renaming it `server` right now was most accurate?